docs/embedded/tutorials/migrate-abs-to-spe.md

```diff
@@ -0,0 +1,333 @@
+---
+title: Sample App to Migrate from Azure Blob Storage container to SharePoint Embedded container
+description: Tutorial in how to migrate from Azure Blob Storage container to SharePoint Embedded container
+ms.date: 07/31/2024
+ms.localizationpriority: high
+---
+
+# Tutorial For Migrating Content From ABS To SPE
+
+## Purpose
+
+This tutorial will guide you through migrating content from Azure Blob Storage (ABS) to SharePoint Embedded (SPE). This is particularly useful for customers who have 500 docs in the blob storage container.
+
+### Prerequisites
+
+1. .NET Core SDK [version 8.0.303](https://dotnet.microsoft.com/en-us/download/dotnet/8.0)
+1. Dotnet environment to run sample app
+    - It can be run Windows, Linux and MacOs
+1. SharePoint Embedded container
+    - For more information how to set up [SPE container](https://aka.ms/start-spe)
+1. Azure Blob Storage container
+    - For more information how to set up [ABS container](../../../../azure/storage/blobs/storage-blobs-introduction)
+
+### Packages
+1. Microsoft Graph SDK (version 5.56.0)
+1. Azure.Identity (version 1.12.0)
+1. Azure.Storage.Blobs (version 12.21.0)
+1. CommandLineParser (version 2.9.1)
+1. Newtonsoft.Json (13.0.3)
+
+## Authentication
+
+### Azure Blob Storage
+
+1. Credentials - Container level Shared Access Signature (SAS) URL.
+1. Permission - Read and List
+
+### SharePoint Embedded site
+
+Minimum Requirements
```

Comment:
I think we can clarify that this is referring to "An app registration in Azure". "App type" could be ambiguous